webscraper/database/init_db.sql

```diff
@@ -0,0 +1,16 @@
+-- Create users table
+CREATE TABLE users (
+    id SERIAL PRIMARY KEY,
+    username VARCHAR(255) NOT NULL UNIQUE,
+    password_hash VARCHAR(255) NOT NULL
```

Comment:
I might have to look into this, it might be in the database where the password gets encrypted and salted with some configuration edits if anyone has an idea?